README.md

```diff
@@ -212,83 +199,57 @@ source_weight: a weight for the source
 z:             a point estimator (when sigmacrit_style="point")
 sc_i:          1/sigma_crit in bins of lens redshift.  The redshift bins
                are defined in "zlvals" config parameter
-z_mean:        expectation value of source redshift
-z_sample:      random sample from source p(z)
+z_mean:        expectation value of source redshift, used for selecting/weighting the source
+z_sample:      random sample from source p(z), used for calculating sigma_crit^-1
 ```
 
 Format of Output Catalog
 ------------------------
-The output catalog has a row for each lens.  For shear_style="reduced",
-ordinary reduced shear style, each row looks like
-```
-    index weight_tot totpairs npair_i rsum_i wsum_i dsum_i osum_i
-```
+The output catalog has a row for each lens.  It contains the same columns regardless of
+input format or styles selected above.
 
-For shear style="lensfit", lensfit style
 ```
-    index weight_tot totpairs npair_i rsum_i wsum_i dsum_i osum_i dsensum_i osensum_i
+    index weight_tot totpairs npair_i rsum_i wsum_i ssum_i dsum_o osum_i dsensum_w_i osensum_w_i dsensum_s_i osensum_s_i 
```

Comment:
I restored the old output format in the shear_units="deltasig" (default) and shear_units="shear" modes.